.travis/check.sh

```diff
@@ -23,6 +27,9 @@ elif [[ -z "$TEST" || "$TEST" == "encoding" ]]; then
     if [[ "$OPENCL" == "yes" ]]; then
         sudo apt-get install fglrx-dev opencl-headers || true
 
+	# --disable-simd and opencl do not work together.  Turn off disable-simd mode.
```

Comment:
This statement is no longer true, right?

Reply:
True, but I think all of travis build scripts get reverted.

We 'could' keep it the way it is (scrap the comments and commented out stuff in the OpenCL), but keep the other, if we want.  then we simply comment out the export line at top of file, and a build will be done without this logic. Then if we need to test, simply uncomment that line and we have a version testing right off.